adcc/Excitation.py

```diff
@@ -42,6 +42,31 @@ def inner(f, kwargs=kwargs):
     return inner
 
 
+def extract_excitation_property_keys(state):
```

Comment:
I get the idea, but it feels a bit complicated to me. Sure there is no simpler way (like using the decorator to automatically build a list of properties or sth like that).

Reply:
Then this would need to be done inside `ExcitedStates` itself. I can certainly do this.

Reply:
Yes I agree! I have no clue what is better. Just try a bit. The problem with python is that the enormous flexibility sometimes lets you buy into too much metaprogramming ...

Reply:
Done. I like the newer version much better.

Reply:
I think it lives in the right spot now 😄